src/evict/evict.h

```diff
@@ -137,6 +137,8 @@ struct __wt_evict {
 #define WT_EVICT_CALL_URGENT 0x4u   /* Urgent eviction */
 /* AUTOMATIC FLAG VALUE GENERATION STOP 32 */
 
+#define WT_EVICT_MAX_WORKERS 20
```

Comment:
Could you add a comment inside api_data.py to note that this number must match the eviciton.threads_max?
E.g.
```
    Config('eviction', '', r'''
        eviction configuration options''',
        type='category', subconfig=[
            Config('threads_max', '8', r'''
                maximum number of threads WiredTiger will start to help evict pages from cache. The
                number of threads started will vary depending on the current eviction load. Each
                eviction worker thread uses a session from the configured session_max''',
                min=1, max=20), # !!! Must match WT_EVICT_MAX_WORKERS
```

Reply:
Ok done!